NitroxClient/Communication/Packets/Processors/PingRenamedProcessor.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using NitroxClient.Communication.Abstract;
+using NitroxClient.Communication.Packets.Processors.Abstract;
+using NitroxClient.MonoBehaviours;
+using NitroxModel.DataStructures.Util;
+using NitroxModel.Logger;
+using NitroxModel.Packets;
+using UnityEngine;
+
+namespace NitroxClient.Communication.Packets.Processors
+{
+    /// <summary>
+    ///     Only syncs pings from beacons for now.
+    /// </summary>
+    public class PingRenamedProcessor : ClientPacketProcessor<PingRenamed>
+    {
+        private readonly IPacketSender sender;
+
+        public PingRenamedProcessor(IPacketSender sender)
+        {
+            this.sender = sender;
+        }
+
+        public override void Process(PingRenamed packet)
+        {
+            Optional<GameObject> obj = NitroxEntity.GetObjectFrom(packet.Id);
+            if (!obj.HasValue)
+            {
+                // Not the object we're looking for.
+                return;
+            }
+            Beacon beacon = obj.Value.GetComponent<Beacon>();
+            if (!beacon)
+            {
+                // This can be ok if origin of ping instance component was not from a beacon (but from signal or other).
+                return;
+            }
+            if (beacon.GetComponent<Player>())
+            {
+                // Skip over beacon component on player GameObjects
+                return;
+            }
```

Comment:
log the ones that are error?

Reply:
None of these have to be errors. And this information doesn't seem useful most of the time. I wish we had some `.Trace()` or just disable logs from this class by default.